Geometry/CommonDetUnit/interface/GeomDetLess.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TrackerGeometryBuilder_GeomDetLess_H
+#define TrackerGeometryBuilder_GeomDetLess_H
+
+#include "DataFormats/TrajectorySeed/interface/PropagationDirection.h"
```

Comment:
Then it seems to me the implication is this header actually belongs in DataFormats/TrajectorySeed.